README.md

```diff
@@ -102,7 +102,7 @@ Dispatch runs on a non-standard https port on minikube since it uses
 NodePort for the ingress controller service. Hence, update the
 Authorization callback URL of your OAuth2 Client App (created by following
 [How to Create OAuth Client App](docs/create-oauth-client-app.md)) from
-`https://dispatch.local/oauth2` to `https://dispatch.local:<port>/oauth2`
+`https://dispatch.local/v1/iam/oauth2` to `https://dispatch.local:<port>/oauth2`
```

Comment:
do you mean `https://dispatch.local:<port>/v1/iam/oauth2` at the second quote?

Reply:
Oh yeah, I missed to change it. thanks :)